app/controllers/courses_controller.rb

```diff
@@ -1,27 +1,26 @@
 class CoursesController < ApplicationController
   before_action :set_course, except: %i(new create show)
-  before_action :authenticate_user!
-  before_action :check_permitted_or_not , only: %i(edit destroy)
+  before_action :check_permitted_or_not, only: %i(edit destroy)
+
+  def new
+    @course = Course.new
+  end
 
   def index
-    @courses = Course.recent_courses.page params[:page]
+    @course = Course.recent_courses.page params[:page]
```

Comment:
add records per page and dont forget to move 5 to settings file and call it herer
```suggestion
    @course = Course.recent_courses.page(params[:page]).per 5
```